ElastosBountyProgram/front-end/src/service/TeamService.js

```diff
@@ -166,6 +166,42 @@ export default class extends BaseService {
         return result
     }
 
+    async closeTeam(teamId){
+        const teamRedux = this.store.getRedux('team')
+        this.dispatch(teamRedux.actions.loading_update(true))
+         const result = await api_request({
+            path: '/api/team/action/close',
+            method: 'post',
+            data: {
+                teamId
+            }
+        })
+         const curTeamDetail = this.store.getState().team.detail
+        curTeamDetail.status = TEAM_STATUS.CLOSED
+         this.dispatch(teamRedux.actions.loading_update(false))
+        this.dispatch(teamRedux.actions.detail_update(curTeamDetail))
+        
+        return result
+    }
+
+    async activeTeam(teamId){
+        const teamRedux = this.store.getRedux('team')
+        this.dispatch(teamRedux.actions.loading_update(true))
+         const result = await api_request({
+            path: '/api/team/action/active',
+            method: 'post',
+            data: {
+                teamId
+            }
+        })
+         const curTeamDetail = this.store.getState().team.detail
```

Comment:
Wrong indentation, also in the function above. Please keep a proper indent level in your code (all of the changes in this file)